simulation/tools/sim_test_performance.cpp

```diff
@@ -5,17 +5,32 @@
  *
  */
 
-#include <Eigen/Dense>
-#include <boost/shared_ptr.hpp>
-#include <cmath>
-#include <iostream>
+#include <pcl/common/common.h>
+#include <pcl/common/transforms.h>
+#include <pcl/console/parse.h>
+#include <pcl/console/print.h>
+#include <pcl/console/time.h>
+#include <pcl/features/normal_3d.h>
+#include <pcl/io/pcd_io.h>
+#include <pcl/memory.h>
+#include <pcl/pcl_config.h>
+#include <pcl/point_types.h>
+#include <pcl/simulation/camera.h>
+#include <pcl/simulation/model.h>
+#include <pcl/simulation/range_likelihood.h>
+#include <pcl/simulation/scene.h>
+#include <pcl/surface/gp3.h>
+
+// define the following in order to eliminate the deprecated headers warning
```

Comment:
Should the warning go inside the header? You're using this in 4 different headers in this PR.

Reply:
I can find docs about VTK_EXCLUDE_STRSTREAM_HEADERS only up to VTK 5.10, and we require 6.2. I don't have VTK checked out locally but I can `git log --follow Common/vtkIOStream.h` ([reference link](https://gitlab.kitware.com/vtk/vtk/-/blob/v5.10.0/Common/vtkIOStream.h#L70-97)) soonish. In case the define's gone, I'll delete this.

Reply:
In the long diff list I missed that this was always present, but thanks for following up on this.

Reply:
[This](https://gitlab.kitware.com/vtk/vtk/-/commit/7765a5f3d7bbdb57595017f21b4331fbe9bc5a53) is the commit removing `VTK_EXCLUDE_STRSTREAM_HEADERS` which is valid from v6.0.0.rc1